habitat_sim/simulator.py

```diff
@@ -391,7 +398,7 @@ def get_observation(self):
                 else:
                     tgt.read_frame_rgba_gpu(self._buffer.data_ptr())
 
-                return self._buffer.flip(0).clone()
```

Comment:
Got it, you moved copying logic to `NoNoise`. Still will copying/cloning be valid for all types of sensors?

Reply:
For all visual sensors, yes. This current sensor class is really for just visual sensors.  When we introduce other sensors, I would imagine those will have their own separate python wrapper.